### ISSUE TITLE
SwiftUI: active scheme does not build this file

### ISSUE THREAD

**Issue by innoreq** [tuist/tuist#2569]
**Describe the bug**
After setting up a project from scratch, and adopting the tuist configuration, the generated project builds perfectly. However, the SwiftUI previews cannot be used; the preview canvas provides an error "active scheme does not build this file".

I compared the original project setup with the manifested one, and I cannot find a significant difference explaining that behaviour (e.g., "Enable Previews" in build settings is "yes").

**Expected behavior**
Previews should work.

**Desktop (please complete the following information):**

- OS: macOS 11.2.2
- Xcode: Version 12.4 (12D4e)

**Comment by innoreq** [comment]
I started with my own project, not with --template. What exactly is different between those two methods?

**Comment by innoreq** [comment]
I reinitialized the tuist environment as mentioned above, took over all manifests content to the new environment, added all sources and frameworks, and tried to build again.

Then, Xcode failed with an error for both local frameworks I'm using, regarding a missing ".h" umbrella file there. 

I fixed this by setting "USE_HEADERMAP" to "No". After that, previews were working.

So, I can continue now, but the question remains to me, what the exact difference between those two methods is.
